src/ActionBmwCodeCell.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { html } from "htm/preact";
+import { BlissSymbol, BciAvId } from "./BlissSymbol";
+import "./ActionBmwCodeCell.scss";
+
+type ActionBmwCodeCellProps = {
+  id: string,
```

Comment:
"id" -> "type"?

Moreover, the argument passed into this component should only be the `options` part starting at line 18, without the `type` part. The `type` value is only used by `Palette` component to instantiate this component.